packages/components/src/button-group/stories/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import Button from '../../button';
+import ButtonGroup from '../';
+
+export default { title: 'Button Group', component: ButtonGroup };
+
+export const _default = () => (
+	<ButtonGroup>
+		<Button isPrimary>Button 1</Button>
+		<Button isPrimary>Button 2</Button>
```

Comment:
I reason that only one of them should be primary:

![Screen Shot 2019-10-11 at 12 51 10](https://user-images.githubusercontent.com/699132/66646214-d1d73d80-ec25-11e9-82a7-4fd66294d422.png)

They look like one big button at the moment.

Reply:
Is there a way to add CSS to Storybook? Also, the buttons when running in dev look different to  than the buttons in production, which makes me think there is CSS loaded somehow

Reply:
https://storybook.js.org/docs/configurations/default-config/#css-support

Reply:
Addressed in #17910